flexmeasures/data/services/scheduling.py

```diff
@@ -233,7 +237,12 @@ def load_custom_scheduler(scheduler_specs: dict) -> Tuple[Callable, str]:
         module, scheduler_specs["function"]
     ), "Module at {module_descr} has no function {scheduler_specs['function']}"
 
-    return getattr(module, scheduler_specs["function"]), source_name
+    if hasattr(module, "__version__"):
+        source_info["version"] = str(module.__version__)
+    if hasattr(module, "__author__"):
+        source_info["name"] = str(module.__author__)
```

Comment:
This is a design choice. Do we want to allow, warn or raise when these have not been set? I have no strong preference, but let's document the choice.

Reply:
I added warnings for now.